bench/postgres/optimized/pg_enum.ll

```diff
@@ -76,11 +76,13 @@ list_length.exit:                                 ; preds = %2, %3
   %19 = trunc nuw nsw i64 %18 to i32
   %20 = shl nuw nsw i64 %18, 3
   %21 = tail call ptr @palloc(i64 noundef %20) #10
-  %.not = icmp eq i32 %19, 0
+  %.not = icmp eq i32 %6, 0
   br i1 %.not, label %.preheader, label %.lr.ph
 
 .lr.ph:                                           ; preds = %._crit_edge
   %22 = getelementptr inbounds i8, ptr %7, i64 64
+  %umax = tail call i32 @llvm.umax.i32(i32 %19, i32 1)
+  %wide.trip.count109 = zext nneg i32 %umax to i64
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597406008